src/20250814_Multi_GHOInkLaunch/GHOInkLaunch_20250814.s.sol

```diff
@@ -0,0 +1,169 @@
+    );
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](1);
+    actions[0] = GovV3Helpers.buildAction(payload0);
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
+  }
+}
+
+contract DeployInk is InkScript {
+  function run() external broadcast {
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory launchActions = new IPayloadsControllerCore.ExecutionAction[](1);
+    launchActions[0] = GovV3Helpers.buildAction(
+      GovV3Helpers.deployDeterministic(type(AaveV3Ink_GHOInkLaunch_20250814).creationCode)
+    );
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(launchActions);
+  }
+}
+
+contract CreateProposal is EthereumScript {
+  function run() external {
+    // create payloads
+    PayloadsControllerUtils.Payload[] memory payloads = new PayloadsControllerUtils.Payload[](6);
+
+    // compose actions for validation
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsEthereum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsEthereum[0] = GovV3Helpers.buildAction(
+      type(Ethereum_Ink_AaveV3GHOLane_20250814).creationCode
+    );
+    payloads[0] = GovV3Helpers.buildMainnetPayload(vm, actionsEthereum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsArbitrum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsArbitrum[0] = GovV3Helpers.buildAction(
+      type(Arbitrum_Ink_AaveV3GHOLane_20250814).creationCode
+    );
+    payloads[1] = GovV3Helpers.buildArbitrumPayload(vm, actionsArbitrum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBase = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBase[0] = GovV3Helpers.buildAction(type(Base_Ink_AaveV3GHOLane_20250814).creationCode);
+    payloads[2] = GovV3Helpers.buildBasePayload(vm, actionsBase);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsAvalanche = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsAvalanche[0] = GovV3Helpers.buildAction(
+      type(Avalanche_Ink_AaveV3GHOLane_20250814).creationCode
+    );
+    payloads[3] = GovV3Helpers.buildAvalanchePayload(vm, actionsAvalanche);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsGnosis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsGnosis[0] = GovV3Helpers.buildAction(
+      type(Gnosis_Ink_AaveV3GHOLane_20250814).creationCode
+    );
+    payloads[4] = GovV3Helpers.buildGnosisPayload(vm, actionsGnosis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsInkLaunch = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsInkLaunch[0] = GovV3Helpers.buildAction(
+      type(AaveV3Ink_GHOInkLaunch_20250814).creationCode
+    );
+    payloads[5] = GovV3Helpers._buildPayload(vm, ChainIds.INK, actionsInkLaunch);
```

Comment:
this should use buildInkPayload

Reply:
Done here https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/ebb87e8a15f5c4d3e6aa82f129caa4c4451fe2cd